app/models/user.rb

```diff
@@ -59,7 +59,8 @@ def self.find_or_create_from_provider_data(provider_data, feature_flag_id:)
 
     if user.persisted?
       unless user.valid?
-        Rails.logger.info("[GAI] User persisted BUT not valid, #{user.errors.full_messages.join(';')}, ID=#{user.id}, UID=#{provider_data.uid}, trying to join account")
```

Comment:
Changing this error message, as it is identical to one further down, so seems to be a cut-and-paste error.